src/backend/app/projects/project_crud.py

```diff
@@ -2211,6 +2211,76 @@ def get_project_tiles(
         )
         log.info(f"Basemap created for project ID {project_id}: {outfile}")
 
+
+        get_project_sync = async_to_sync(get_project)
+        project = get_project_sync(db, project_id)
+
+        from app.s3 import add_file_to_bucket
```

Comment:
Try not to import inline like this, it goes against the PEP8 style guide. Always import at the top of the file 👍